### PR TITLE
Material themed tooltips and popups for bug #275

### PR DESCRIPTION
**Pull request by GradyD**
http://www.google.com/design/spec/components/tooltips.html#tooltips-usage

### CODE REVIEW

File: less/_popups.less [hunk 1]
```diff
@@ -1,25 +1,19 @@
 .popover, .tooltip-inner {
-    background: #323232;
-    color: #FFF;
-    border-radius: 2px;
-
+  color: #ececec;
```
Comment (FezVrasta) [inline less/_popups.less]: may you use color variables instead? Feel free to add them to _variables.less, just be sure to give them a meaningful name.

**Comment by GradyD** [comment]
Can do, just do it for `color:` or do it for the background and border also?

**Comment by FezVrasta** [comment]
I'd like to use variables wherever possible (and where it makes sense).

**Comment by GradyD** [comment]
How does this look? I could move a few others but I do not see any of the remaining CSS code reused anywhere.

**Comment by GradyD** [comment]
Note that in this push I modified material.less to try and fix #497. I just moved the _shadows.less include to the bottom of the include statements.

**Comment by FezVrasta** [comment]
I've implemented it because this PR was outdated

**Pull request closed without merging**